npm-shrinkwrap.json

```diff
@@ -1464,9 +1464,9 @@
       "dev": true
```

Comment:
For whatever reason, the `npm-shrinkwrap.json` file is "unstable":
```
$ git checkout reduce-bluebird
$ rm -rf node_modules && npm i
$ git status
On branch reduce-bluebird
Changes not staged for commit:
	modified:   npm-shrinkwrap.json
```

Reply:
It's probably fsevents weirdness as it causes headaches, and because I have to regenerate the npm-shrinkwrap on every single rebase it makes it infeasible to keep track of if it has done weirdness every single time I have to rebuild it because it happens way too often